### PR TITLE
feat: Add Dokan UIView  :dizzy:

### PR DESCRIPTION
**Pull request by claramounir**

### CODE REVIEW

File: Dokan/UIView.swift [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+//  UIView.swift
+//  Dokan
+//
+//  Created by Clara Mounir Adly on 04/08/2022.
+//
+
+import Foundation
+import UIKit
+
+class BaseUIView: UIView {
```
Comment (ahmdmhasn) [inline Dokan/UIView.swift]: Let's make the name more descriptive like `UIViewFromNib`

**Review by MarwaEbrahem — APPROVED** [review]

**Review by ziadali22 — APPROVED** [review]

**Review by oradwanomar — APPROVED** [review]

File: UIDokan/Sources/UIDokan/Style/UIButton+Style.swift [hunk 2]
```diff
@@ -119,6 +119,10 @@ extension UIButton {
     }
 }
 
+//  MARK: - Border Buttons style
+
+//
+
```
Comment (ahmdmhasn) [inline UIDokan/Sources/UIDokan/Style/UIButton+Style.swift]:
```suggestion
```

File: Dokan/UIView.swift [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+//  UIView.swift
+//  Dokan
+//
+//  Created by Clara Mounir Adly on 04/08/2022.
+//
+
+import Foundation
+import UIKit
+
+class DokanUIView: UIView {
```
Comment (ahmdmhasn) [inline Dokan/UIView.swift]:
```suggestion
class UIViewFromNib: UIView {
```

**Review by ahmdmhasn — COMMENTED** [review]
Please move the file to UIDokan since it's reusable.

**Pull request merged**